conans/requirements_dev.txt

```diff
@@ -3,3 +3,4 @@ parameterized>=0.6.3
 mock>=1.3.0, <1.4.0
 WebTest>=2.0.18, <2.1.0
 bottle
+meson>=0.56.0rc2
```

Comment:
That might be too much, to introduce meson as a dev dependency, specially if it doesn't install in some platforms, this will block development.

We would need the new testing framework that would allow to run tests that use different tools installed in the system in a more controlled way.

Reply:
so, to clarify, what exactly should I do?

Reply:
Run the test only if the meson you need is in the system, otherwise skip the test. But do not force that requirement in ``requirements_dev.txt``

Reply:
okay, so tests won't run on CI as I need very recent meson. I'll remove a requirement.

Reply:
okay, removed meson from `requirements_dev.txt`. added check in tests to verify if meson installed and if it has a correct version (0.56.0+).